packages/web-components/src/components/va-textarea/va-textarea.scss

```diff
@@ -25,4 +25,19 @@
 
 .usa-hint {
   display: block;
+}
+
+// Overrides for when label is wrapped with a header
+h1, h2, h3, h4, h5, h6 {
+  .usa-label {
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+
+    .usa-label--required {
+      font-family: "Source Sans Pro Web", "Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans;
```

Comment:
We have css properties for font families - https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/dist/tokens/css/variables.css#L127

Would it be possible to use those here?

Reply:
Technically, yes, but I'd have to re-define those variables within the va-textarea.scss file because those variables aren't available in Storybook yet.

Reply:
Variables are imported into the web-components package here:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/global/main.css#L1